w: 1280.0
h: 800.0
f: 700.0
k: 1.5
u_min: -640.0
u_max: 640.0
v_min: -400.0
v_max: 400.0
cameraMatrix:
  rows: 3
  cols: 3
  data: [700, 0,   640,
         0,   700, 400,
         0,   0,     1]
distCoeffs:
  rows: 1
  cols: 4
  data: [0, 0, 0, 0]
pts_m:
  rows: 4
  cols: 3
  data: [-0.15, -0.15, 0,
          0.15, -0.15, 0,
          0.15,  0.15, 0,
         -0.15,  0.15, 0]
